kyo-core/jvm/src/test/scala/kyoTest/coreBytecodeSizeTest.scala

```diff
@@ -38,7 +38,7 @@ class coreBytecodeSizeTest extends KyoTest:
         assert(map == Map(
             "test"        -> 28,
             "resultLoop"  -> 112,
-            "handleLoop"  -> 280,
+            "handleLoop"  -> 264,
```

Comment:
There are fewer bytecodes to invoke `handler.accepts` than to perform the equality check.